<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gragh_show</class>
 <widget class="QWidget" name="Gragh_show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="1,4">
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>实验信息</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,8">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>存储地址</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="Folder_lineEdit"/>
           </item>
           <item>
            <widget class="QToolButton" name="Folder_Button">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>采样设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,4">
           <property name="verticalSpacing">
            <number>9</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>进样时长(s)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>流量(L/min)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="Flowtime_spinBox">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>清洗时长(s)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="Inputtime_spinBox">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="value">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="Cleartime_spinBox">
             <property name="focusPolicy">
              <enum>Qt::WheelFocus</enum>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="value">
              <number>60</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>传感器列表</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableView" name="Senser_stableView">
             <property name="minimumSize">
              <size>
               <width>226</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>226</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="Dataclear_Button">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>数据清洗</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Collectbegin_Button">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>开始采集</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Pause_Button">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>暂停采集</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Save_Button">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>保存数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Line_Show">
        <attribute name="title">
         <string>曲线图</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="Linegragh_Layout"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/icon.qrc"/>
 </resources>
 <connections/>
</ui>
